app/views/admin/users/_form.html.erb

```diff
@@ -11,82 +10,131 @@
     </div>
   <% end %>
 
-  <div class="form-group">
+  <h3>General information</h3>
+
+  <div class="form__group">
     <%= f.label :first_name %>
-    <%= f.text_field :first_name %>
+    <div class="form__field">
+      <%= f.text_field :first_name %>
+    </div>
   </div>
 
-  <div class="form-group">
+  <div class="form__group">
     <%= f.label :last_name %>
-    <%= f.text_field :last_name %>
+    <div class="form__field">
+      <%= f.text_field :last_name %>
+    </div>
   </div>
 
-  <div class="form-group">
-    <%= f.label :email %>
-    <%= f.email_field :email %>
+  <div class="form__group">
+    <%= f.label :agency_id %>
+    <div class="form__field">
+      <%= f.collection_select :agency_id, Agency.all, :id, :name %>
+    </div>
   </div>
 
-  <% if !@user.persisted? %>
-    <div class="field">
-      <%= f.label :password %><br />
-      <%= f.password_field :password, autocomplete: "off" %>
-      <% if @minimum_password_length %>
-        <br />
-        <em><%= @minimum_password_length %> characters minimum</em>
-      <% end %>
+  <div class="form__group">
+    <%= f.label :role_id %>
+    <div class="form__field">
+      <%= f.collection_select :role_id, Role.all, :id, :name %>
     </div>
+  </div>
 
-    <div class="field">
-      <%= f.label :password_confirmation %><br />
-      <%= f.password_field :password_confirmation, autocomplete: "off" %>
+  <%= f.collection_check_boxes(:expertise_ids, Expertise.all, :id, :name) do |b| %>
```

Comment:
I think this section could really use a grouping label like "Expertise", because as is is now at root level I'm afraid I got the impression I'm being asked if I'm a Bonobo :monkey: :smile: